src/components/verification-grid/verification-grid.ts

```diff
@@ -346,6 +330,30 @@ export class VerificationGridComponent extends AbstractComponent(LitElement) {
         break;
       }
 
+      // if the user is holding down both ctrl and D, we should remove the
+      // current selection (this is a common behavior in other applications)
+      case "d": {
+        if (event.ctrlKey) {
+          event.preventDefault();
+          this.removeSubSelection();
+        }
+        break;
+      }
+
+      case "a": {
+        if (event.ctrlKey) {
+          // we prevent default on the ctrl + A event so that chrome doesn't
+          // select all the text on the page
+          event.preventDefault();
+        }
+
+        if (canSubSelect) {
+          this.subSelectAll();
+        }
```

Comment:
when can we ever not sub-select?

Reply:
When there is one tile in the verification grid

This was a requested change from our first round of user testing

Reply:
Sure. Perhaps move it to the selection functions.
My gripe here is that is looked like special casing. Rather than remember to do it everywhere let's just let the selection functions take care of choosing whether or not they have an affect?

Reply:
Fixed in commit: [223bf99759deaff9c0141e82a0aa515a6e7ff2ba](https://github.com/ecoacoustics/web-components/commit/223bf99759deaff9c0141e82a0aa515a6e7ff2ba)